### PR TITLE
DepMapCCLE and minor Single cell description

### PR DESCRIPTION
**Pull request by Ayush-Praveen**
Updated Doc for DepMapCCLE and Single cell. Single Cell app is currently being redesigned, so I have added minimal description for it.

### CODE REVIEW

File: docs/Data Lake.md [hunk 1]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
 
+##DepMap CCLE
+
+###Opening the app
+
+Upon opening the Discover portal on Polly, choose the Depmap data repository. The page should look something like this.
+
+![Selecting app](../img/DepMapCCLE/image1.png)
+
+After selecting the Depmap repository, you’ll be able to see a dashboard and the DepMap CCLE app icon. The dashboard will give you an overview of the repository, for instance, the information of different cell lines in the  repository, distribution of tissue lineages, metastatic status of samples, etc. among the datasets.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:166]: Depmap repository in italics
....along with the DepMap CCLE *application (Italics)

File: docs/Data Lake.md [hunk 2]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+![Kibana dashboard](../img/DepMapCCLE/image2.png)
+
+Click on the below icon to start the DepMap CCLE app.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:170]: Italics

File: docs/Data Lake.md [hunk 3]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+![DepMap icon](../img/DepMapCCLE/image3.png)
+
+The app will open and you should see the home page which contains two examples describing in brief  usage of the application.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:174]: Once inside the application, you can find two examples on the home page that briefly describes the usage of the application

File: docs/Data Lake.md [hunk 4]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+![App overview](../img/DepMapCCLE/image4.png)
+
+###Exploring the DepMap CCLE data lake
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:178]: Remove a hash

File: docs/Data Lake.md [hunk 5]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+The app provides the functionality to explore the DepMap CCLE data lake. We can query the datalake for three different functionalities for different genes in the application. Each functionality has a separate tab in left for easy access.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:180]: *This application........... 
*You can query the datalake
*...on the left for easy access

File: docs/Data Lake.md [hunk 6]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+The app provides the functionality to explore the DepMap CCLE data lake. We can query the datalake for three different functionalities for different genes in the application. Each functionality has a separate tab in left for easy access. 
+
+*   Gene Essentiality
+
+*   Gene Cluster
+
+*   CCLE Heatmap
+
+####Gene Essentiality
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:188]: Remove a hash

File: docs/Data Lake.md [hunk 7]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+####Gene Essentiality 
+
+Gene Essentiality of the gene describes the efficacy vs selectivity map for a gene based on different models. The terms can be defined as: 
+
+*   **Efficacy** : how essential the gene is in the sensitive cell lines (the more negative the efficacy is, the more essential the gene is).
+
+*   **Selectivity** : how selectively essential the gene is between sensitive and resistant cell lines (the more positive the selectivity is, the more selective the gene is).
+
+As shown in the following image, one can search for a gene for it’s essentiality and press Go to check the scores.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:196]: ..*You can search 
. Press *Go (Italics)

File: docs/Data Lake.md [hunk 8]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+*   **Efficacy** : how essential the gene is in the sensitive cell lines (the more negative the efficacy is, the more essential the gene is).
+
+*   **Selectivity** : how selectively essential the gene is between sensitive and resistant cell lines (the more positive the selectivity is, the more selective the gene is).
+
+As shown in the following image, one can search for a gene for it’s essentiality and press Go to check the scores. 
+
+![Select gene](../img/DepMapCCLE/image5.png)
+
+The search returns a Essentiality map in the left with the searched gene highlighted with Red. The right panel describes the efficacy and selectivity scores of the matched gene and the dependency scores in different models in the datalake.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:200]: *an essentiality map (Italics).....*on the left *panel............... with *red color. 
..of the matched genes(italics) and dependency scores(Italics)
...scores *of different models

File: docs/Data Lake.md [hunk 9]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+The search returns a Essentiality map in the left with the searched gene highlighted with Red. The right panel describes the efficacy and selectivity scores of the matched gene and the dependency scores in different models in the datalake. 
+
+![Essentiality](../img/DepMapCCLE/image6.png)
+![Matched genes](../img/DepMapCCLE/image7.png)
+![Dep score](../img/DepMapCCLE/image8.png)
+
+####Gene Cluster
+
+In concept, genes that work together as complexes or pathways should show similar dependency scores across cell lines. With this assumption, 2,492 essential genes were clustered based on the similarity of their dependency scores across 423 cell lines—the resulting clusters group essential genes that likely work together.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:208]: ...that * are likely *to work together.

File: docs/Data Lake.md [hunk 10]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+As shown in the figure below one can query for their gene of interest to see what genes cluster together with it. To do the search, type in the name of the gene, suggestions come up to help you select the gene of your interest. There is an option to change the cluster size to change the parameters for stringency of clustering. Small cluster equate to stringent cutoffs.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:210]: ...*you can query for *your gene of interest to see *which genes ....
..type in the name of the gene*.  Suggestions *will be displayed to help you .....
*cluster size (italics)

File: docs/Data Lake.md [hunk 11]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+####Gene Cluster
+
+In concept, genes that work together as complexes or pathways should show similar dependency scores across cell lines. With this assumption, 2,492 essential genes were clustered based on the similarity of their dependency scores across 423 cell lines—the resulting clusters group essential genes that likely work together.
+
+As shown in the figure below one can query for their gene of interest to see what genes cluster together with it. To do the search, type in the name of the gene, suggestions come up to help you select the gene of your interest. There is an option to change the cluster size to change the parameters for stringency of clustering. Small cluster equate to stringent cutoffs.  
+
+![Cluster gene select](../img/DepMapCCLE/image9.png)
+
+The query returns a set of efficacy vs selectivity plot of the gene, a t-SNE plot for functional similarity and genes clustered with the searched gene in the form of a table, connectivity map and correlation scores.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:214]: Remove a set of  
*an efficacy vs selectivity plot(Italics) 
*t-SNE Plot (Italics)
....similarity *and the clustered genes(italics) with the searched gene in the tabular form along with the Connectivity map(Italics) and Correlation scores(Italics)

File: docs/Data Lake.md [hunk 12]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+The query returns a set of efficacy vs selectivity plot of the gene, a t-SNE plot for functional similarity and genes clustered with the searched gene in the form of a table, connectivity map and correlation scores. 
+
+![Eff plot](../img/DepMapCCLE/image10.png)
+
+![tSNE](../img/DepMapCCLE/image11.png)
+
+![Clustered genes](../img/DepMapCCLE/image12.png)
+
+####CCLE Heatmap
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:222]: Remove hash

File: docs/Data Lake.md [hunk 13]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+CCLE heatmap aids a user in visualizing the cancer specific expression of selected genes in different cell lines models along with the dependency scores of different genes. The heatmap requires three inputs, **Cancer Type** which will help to select the cell lines models, **Dependency Genes** for dependency scores of genes of interest and **Genes for Expression** which will display the expression of the genes. The inputs can be passed as shown in the following figure.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:224]: CCLE*Heatmap(Italics)
Cancer Types, Dependency Genes, and Genes of expression should be in Italics

File: docs/Data Lake.md [hunk 14]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+![Clustered genes](../img/DepMapCCLE/image12.png)
+
+####CCLE Heatmap
+
+CCLE heatmap aids a user in visualizing the cancer specific expression of selected genes in different cell lines models along with the dependency scores of different genes. The heatmap requires three inputs, **Cancer Type** which will help to select the cell lines models, **Dependency Genes** for dependency scores of genes of interest and **Genes for Expression** which will display the expression of the genes. The inputs can be passed as shown in the following figure. 
+
+![Select cancer](../img/DepMapCCLE/image13.png)
+
+Clicking on the plot button processes the data and the result of the query is a heatmap with columns as the sample, rows for the genes and column descriptors describing the dependency scores of the genes.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:228]: *Plot (Italics)

File: docs/Data Lake.md [hunk 15]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+![Select cancer](../img/DepMapCCLE/image13.png)
+
+Clicking on the plot button processes the data and the result of the query is a heatmap with columns as the sample, rows for the genes and column descriptors describing the dependency scores of the genes. 
+
+![Heatmap](../img/DepMapCCLE/image14.png)
+
+##scViz
+
+scViz provides up a visualization interface for single cell data hosted on GEO. The datalake working on it's backend is updated regularly with new datasets. The app has different visualizations for understanding data - Dimensionality reduction plot, Violin plot, Dot Plot.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:234]: ..provides *a visualization (Remove up)
-Dimensionality reduction plot,violin plot , dot plot should be in italics

File: docs/Data Lake.md [hunk 16]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
 
 ![Preliminary hitlist](../img/Discover/image13.png)
 
+##DepMap CCLE
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md:158]: @Shreya-crypto Have shifted the Polly discover and DepMap CCLE under discover apps and with that case it should be Heading 2 only.

File: docs/Data Lake.md [hunk 17]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+###Opening the app
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md:160]: With same logic heading 3 is acts for further elements.

File: docs/Data Lake.md (hunk 4 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:178]: Explained the logic of making it heading 3 above

File: docs/Data Lake.md (hunk 3 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:174]: changed

File: docs/Data Lake.md (hunk 2 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:170]: changed

File: docs/Data Lake.md (hunk 1 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:166]: changed

File: docs/Data Lake.md [hunk 18]
```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
...
+
+Upon opening the Discover portal on Polly, choose the Depmap data repository. The page should look something like this.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md:162]: changed

File: docs/Data Lake.md (hunk 5 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:180]: changed

File: docs/Data Lake.md (hunk 6 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:188]: Explained the logic above

File: docs/Data Lake.md (hunk 7 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:196]: changed

File: docs/Data Lake.md (hunk 8 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:200]: changed

File: docs/Data Lake.md (hunk 9 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:208]: It's a definite case when we are talking about 'genes that work together as complexes or pathways', there is no probable case here. the probably case is with dependency scores and for that i have added 'similar'

File: docs/Data Lake.md (hunk 12 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md:222]: Explained the logic above

File: docs/Data Lake.md (hunk 15 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: changed

File: docs/Data Lake.md (hunk 14 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: changed

File: docs/Data Lake.md (hunk 13 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: changed

File: docs/Data Lake.md (hunk 10 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: changed

File: docs/Data Lake.md (hunk 9 above)
Comment (Shreya-crypto) [reply docs/Data Lake.md:208]: Got it but the word likely itself is used to express probability

File: docs/Data Lake.md (hunk 11 above)
Comment (Shreya-crypto) [reply docs/Data Lake.md]: ?

Comment (Ayush-Praveen) [reply docs/Data Lake.md]: changed already

**Pull request merged**